compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -308,30 +277,61 @@ pub fn check_tied_features(
 /// Used to generate cfg variables and apply features
 /// Must express features in the way Rust understands them
 pub fn target_features(sess: &Session, allow_unstable: bool) -> Vec<Symbol> {
```

Comment:
There is a very similar function in `compiler/rustc_codegen_gcc/src/lib.rs` which now works quite differently from its LLVM counterpart...
Cc @rust-lang/wg-gcc-backend

Reply:
Thanks. I plan at some point to try to remove duplications between the 2 codegens.